contracts/scripts/test.sh

```diff
@@ -1,15 +1,35 @@
 #!/bin/bash
 
-set -euo pipefail
+# To run this:
+#
+# .github/scripts/run_consensus.sh
+# source contracts/env/dev
+# contracts/scripts/deploy.sh
+# contracts/scripts/test.sh
 
-E2E_PATH=$(pwd)/e2e-tests
-CONTRACTS_PATH=$(pwd)/contracts
-
-# shellcheck source=contracts/scripts/test_env.sh
-. "$CONTRACTS_PATH/scripts/test_env.sh"
+set -euox pipefail
```

Comment:
Can we check somewhere at the beginning here that the `${PWD}` is root directory of the `aleph-node` project? Otherwise it will not work.